languages/tree-sitter-stack-graphs-typescript/README.md

```diff
@@ -0,0 +1,38 @@
+# tree-sitter-stack-graphs definition for TypeScript
+
+This project defines tree-sitter-stack-graphs rules for TypeScript using the [tree-sitter-typescript](https://www.npmjs.com/package/tree-sitter-typescript) grammar.
+
+## Development
+
+The project is organized as follows:
+
+- The stack graph rules are defined in `src/stack-graphs.tsg`.
+- Builtins sources and configuration are defined in `src/builtins.ts` and `builtins.cfg` respectively.
+- Tests are put into the `test` directory.
+
+Make sure all development dependencies are installed by running:
+
+    npm install
+
+Run all tests in the project by executing the following:
+
+    npm test
```

Comment:
Is this still the preferred way to run the tests? I've been running them locally with `cargo test`, which is how I see the CI job does it as well.